ui/components/component-library/banner-alert/README.mdx

```diff
@@ -12,8 +12,6 @@ import { BannerBase } from '..';
 
 ## Props
 
-The `BannerAlert` accepts all props below as well as all [Box](/docs/components-ui-box--default-story#props) component props
-
 <ArgsTable of={BannerAlert} />
 
 The `BannerAlert` accepts all `BannerBase` component props below
```

Comment:
should this be removed as well? was removed in another PR [here](https://github.com/MetaMask/metamask-extension/pull/20538/commits/8209974abb8b69e935365a3c7c13a58351fcf291)

Reply:
Correct! This can be removed as well because the props will show up in the ArgsTable above